internal/herd_catalogue_regression_test.ml

```diff
@@ -266,7 +266,16 @@ let () =
     | "show" :: [] -> show_tests ?j ?timeout flags
     | "test" :: [] -> run_tests ?j ?timeout flags
     | "promote" :: [] -> promote_tests ?j flags
```

Comment:
Am I mistaken or all calls to `herd_catalogue_regression_test` are with `nopromote`? In that case, should we just put a warning or a failure with `promote`?